_plugins/build.rb

```diff
@@ -0,0 +1,7 @@
+`npm install`
+
+production = ENV['BRANCH'] == 'master' or ENV['BRANCH'] == 'staging'
```

Comment:
Use `||` instead of `or`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.